src/thoas.erl

```diff
@@ -11,7 +11,8 @@
 ]).
 
 -type decode_options() :: #{
-    strings => reference | copy
+    strings => reference | copy,
+    keys => to_existing_atom | to_atom
```

Comment:
I think this is missing `reference | copy`?

Reply:
Great catch! You are right.